src/assets/buildFonts.js

```diff
@@ -5,11 +5,19 @@ const path = require('path');
 
 function buildFonts(conf) {
 
-  const fontSrc = path.join(conf.themeConfig.root, conf.themeConfig.fonts.src, '**', '*.{eot,ttf,woff,woff2,otf,svg}');
-  const fontDest = path.join(conf.themeConfig.root, conf.themeConfig.fonts.dest);
+  let fontsDirs = conf.themeConfig.fonts.src;
```

Comment:
We should be able to strip all of this repetition out with https://www.npmjs.com/package/arraywrap, thoughts?